install/scripts/install-cmake.sh

```diff
@@ -0,0 +1,10 @@
+export INSTALL_PREFIX=${INSTALL_PREFIX:-/opt/cmake}
```

Comment:
this script is now unused but might be handy in the future.
shall I keep or remove it?